ui/app/js/controllers/rules_controller.js

```diff
@@ -321,6 +321,49 @@ function($scope, $routeParams, $location, $timeout, Rules, Search, $modal, $rout
     });
   };
 
+  $scope.openNewScheduledDuplicateModal = function(rule) {
+
+    var modalInstance = $modal.open({
+      templateUrl: 'rule_scheduled_change_modal.html',
+      controller: 'NewRuleScheduledChangeCtrl',
+      size: 'lg',
+      backdrop: 'static',
+      resolve: {
+        scheduled_changes: function() {
+          return [];
+        },
+        sc: function() {
+          if(rule.scheduled_change !== null && rule.scheduled_change.change_type === "insert") {
+            sc = angular.copy(rule.scheduled_change);
+            delete sc.sc_id;
+            delete sc.sc_data_version;
+            delete sc.when;
+            delete sc.complete;
+            delete sc.change_type;
```

Comment:
I think you also need the bits in the "else" block here, to fix the problem noted above.